### PR TITLE
Update to 0.0.21

### PR DESCRIPTION
**Pull request by dhirschfeld**

### CODE REVIEW

**Comment by dhirschfeld** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {% set name = "prometheus_client" %}
-{% set version = "0.0.15" %}
+{% set version = "v0.0.21" %}
```
Comment (isuruf) [inline recipe/meta.yaml]: conda package version shouldn't have `v` in it

**Review by isuruf — COMMENTED** [review]
Can this recipe be `noarch: python`?

**Comment by dhirschfeld** [comment]
> *Can this recipe be `noarch: python`?*

Possibly, I'm reluctant to try that though as I know conda has recently has problems with `noarch: python` on Windows so, given that this works as-is I'd prefer to punt on that one until the next release by which time conda would hopefully have fixed the problem. Maybe they have already fixed it but if not, any effort moving to `noarch: python` now would be wasted.

**Comment by dhirschfeld** [comment]
***xref:*** https://github.com/conda/conda/issues/6057#issuecomment-342679848 & https://github.com/conda/conda/pull/6277

**Comment by dhirschfeld** [comment]
@isuruf - could you please merge this in since all the tests have passed. It would be good to have the latest version available for conda users. I'm interested in particular as it's a new dependency for [JupyterHub](https://jupyterhub.readthedocs.io/en/latest/) which I'm using.

**Comment by isuruf** [comment]
I don't have access. ping @jessicaaustin, @kwilcox

**Comment by kwilcox** [comment]
Clean commit history next time please! 😛

**Comment by dhirschfeld** [comment]
I figured you'd choose the squash-and-merge option when you pulled the trigger so didn't bother cleaning it up myself.

**Pull request merged**